apps-api/backoffice/src/modules/election/models.ts

```diff
@@ -1,13 +1,24 @@
 import {
   ElectionCandidate,
   ElectionInfo,
+  ElectionStatus,
   FilePath,
   ImageFileMimeType,
 } from '@pple-today/api-common/dtos'
 import { EligibleVoterType } from '@pple-today/database/prisma'
 import { Static, t } from 'elysia'
 
-export const ListElectionResponse = t.Array(ElectionInfo)
+export const ListElection = t.Intersect([
```

Comment:
Maybe changing to this name to have more details

```suggestion
export const ElectionWithCurrentStatus = t.Intersect([
```